packages/experiments/src/components/Form/inputs/dropdown/FormDropdown.tsx

```diff
@@ -33,20 +61,56 @@ export class FormDropdown extends FormBaseInput<
    * Render a Fabric Dropdown
    */
   public render(): JSX.Element {
+    const { currentValue } = this.state;
+
     return (
       <Dropdown
         options={[]}
         {...this.props.dropdownProps}
         // These props cannot be overridden
         key={this.props.inputKey}
         onChanged={this._onChanged}
-        selectedKey={this.state.currentValue}
+        {...(currentValue && Array.isArray(currentValue)
+          ? { selectedKeys: currentValue }
+          : { selectedKey: currentValue })}
       />
     );
   }
 
   private _onChanged = (option: IDropdownOption): void => {
-    this.setValue(option.key);
+    const { dropdownProps } = this.props;
+
+    if (dropdownProps !== undefined && dropdownProps.multiSelect === true) {
+      const selected = option.selected === true;
+
+      this.setState((prevState: IFormBaseInputState<number | string | number[] | string[]>) => {
```

Comment:
This needs to be a setValue call, so that the Base input class can perform other operations (I think validation is one of these)